openshift/templates/email-verification-agent/email-verification-agent-deploy.yaml

```diff
@@ -4,14 +4,59 @@ apiVersion: v1
 metadata:
   name: ${NAME}
```

Comment:
Make sure ${SUFFIX} is applied consistently across the templates.  There are many places where it has not been applied.  References/connections between resources are established by names and/or labels.  If the naming is not consistent resources don't get wired up correctly.